keras_cv/models/backbones/efficientnet_v1/efficientnet_v1_backbone.py

```diff
@@ -0,0 +1,484 @@
+            excite ratios passed to the squeeze and excitation blocks.
+
+    Usage:
+    ```python
+    # Construct an EfficientNetV1 from a preset:
+    efficientnet = keras_cv.models.EfficientNetV1Backbone.from_preset(
+        "efficientnetv1_b0"
+    )
+    images = tf.ones((1, 256, 256, 3))
+    outputs = efficientnet.predict(images)
+
+    # Alternatively, you can also customize the EfficientNetV1 architecture:
+    model = EfficientNetV1Backbone(
+        stackwise_kernel_sizes=[3, 3, 5, 3, 5, 5, 3],
+        stackwise_num_repeats=[1, 2, 2, 3, 3, 4, 1],
+        stackwise_input_filters=[32, 16, 24, 40, 80, 112, 192],
+        stackwise_output_filters=[16, 24, 40, 80, 112, 192, 320],
+        stackwise_expansion_ratios=[1, 6, 6, 6, 6, 6, 6],
+        stackwise_strides=[1, 2, 2, 2, 1, 2, 1],
+        stackwise_squeeze_and_excite_ratios=[
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+        ],
+        width_coefficient=1.0,
+        depth_coefficient=1.0,
+        include_rescaling=False,
+    )
+    images = tf.ones((1, 256, 256, 3))
+    outputs = efficientnet.predict(images)
+    ```
+    """  # noqa: E501
+
+    def __init__(
+        self,
+        *,
+        include_rescaling,
+        width_coefficient,
+        depth_coefficient,
+        dropout_rate=0.2,
+        drop_connect_rate=0.2,
+        depth_divisor=8,
+        activation="swish",
+        input_shape=(None, None, 3),
+        input_tensor=None,
+        id_skip,
+        stackwise_kernel_sizes,
+        stackwise_num_repeats,
+        stackwise_input_filters,
+        stackwise_output_filters,
+        stackwise_expansion_ratios,
+        stackwise_strides,
+        stackwise_squeeze_and_excite_ratios,
+        **kwargs,
+    ):
+        img_input = utils.parse_model_inputs(input_shape, input_tensor)
+
+        x = img_input
+
+        if include_rescaling:
+            # Use common rescaling strategy across keras_cv
+            x = layers.Rescaling(1.0 / 255.0)(x)
+
+        x = layers.ZeroPadding2D(
+            padding=correct_pad(x, 3), name="stem_conv_pad"
+        )(x)
+
+        # Build stem
+        stem_filters = round_filters(
+            filters=stackwise_input_filters[0],
+            width_coefficient=width_coefficient,
+            divisor=depth_divisor,
+        )
+        x = layers.Conv2D(
+            filters=stem_filters,
+            kernel_size=3,
+            strides=2,
+            padding="valid",
+            use_bias=False,
+            kernel_initializer=conv_kernel_initializer(),
+            name="stem_conv",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=3,
+            name="stem_bn",
+        )(x)
+        x = layers.Activation(activation, name="stem_activation")(x)
+
+        # Build blocks
+        block_id = 0
+        blocks = float(
+            sum(num_repeats for num_repeats in stackwise_num_repeats)
```

Comment:
Why not

```python
sum(stackwise_num_repeats)
```

Reply:
True. Both gives same result while one giving in shorter code.